src/lib/model/index.js

```diff
@@ -13,6 +13,8 @@
 
 const InboundTransfersModel = require('./InboundTransfersModel.js');
 const OutboundTransfersModel = require('./OutboundTransfersModel.js');
+const OutboundBulkQuotesModel = require('./OutboundBulkQuotesModel');
+const OutboundBulkTransfersModel = require('./OutboundBulkTransfersModel.js');
```

Comment:
I would favour an approach where outbound bulk transfers are modelled the same way as outbound single transfers; i.e. in a single state machine model. Is there a reason to separate them?

Reply:
@bushjames This is a temporary design pending availability of bulk account lookup. I had earlier implemented this as a single state machine model but it was later agreed that the implementation be broken up.